07/reshape-a-matrix.instructor.md

```diff
@@ -13,18 +13,25 @@ def reshape_matrix(matrix, r, c):
     originalRows = len(matrix)
     originalColumns = len(matrix[0])
     
+    # if rows * cols of original matrix is not 
+    # equal to given r * c
+    # not possible to transform matrix into desired shape 
     if originalRows * originalColumns != r * c:
+        # return original matrix
         return matrix
-    
-    new_row = []
-    new_matrix = []
-    for i in range(originalRows):
-        for j in range(originalColumns):
-            new_row.append(matrix[i][j])
 
-            if len(new_row) == c:
-                new_matrix.append(new_row)
-                new_row = []
```

Comment:
As mentioned in Slack, this solution actually works with the new test case, and was updated by Kelsey in [this commit](https://github.com/Ada-Developers-Academy/core-pses/commit/efb15eceb2ddfd16f1995077387b4cbcb2f6a4fd). However, the solution in the checkpoint _was_ still incorrect but you fixed it.

I'm not too particular that we need to keep both solutions, but I'm pointing out for visibility!